kafka-event-listener/src/main/java/org/hypertrace/core/kafka/event/listener/KafkaLiveEventListener.java

```diff
@@ -73,17 +84,17 @@ public Builder<K, V> withExecutorService(
 
     public KafkaLiveEventListener<K, V> build(
         String consumerName, Config kafkaConfig, Consumer<K, V> kafkaConsumer) {
-      assertCallbacksPresent();
+      Queue<BiConsumer<? super K, ? super V>> callbacks;
+      if (deprecatedCallbacksFlow.size() > 0) {
```

Comment:
Do we need these size check here? if nothing was added to it, then it's already an empty queue.